include/tracing_impl/lttng.h

```diff
@@ -28,11 +32,10 @@
  * tracing output, and arguments <arg1> and <arg2> with <name1> and
  * <name2> will appear in that trace as data.
  *
+ * Add a macro to the top level tracing.h
```

Comment:
I don't see a file called `tracing.h` anymore. Did you mean `nccl_ofi_tracepoint.h`?